docs/declarative-customization/column-formatting.md

```diff
@@ -816,7 +816,100 @@ Both the example uses defaultHoverField
 ```
 
 
+## Column formatter reference
 
+Users can refer to a column's formatter JSON inside another column (or view formattter) and use it along with other elements to build a custom column visualization. This can be done by using `columnFormatterReference` property.
```

Comment:
Users can refer to a column's formatter JSON inside another column/view formatter and use it along with other elements to build a custom column visualization. This can be done by using `columnFormatterReference` property.